### PR TITLE
ci: Add cache for kernel experimental

### LINKED ISSUE #1973: add cache for kernel experimental

### PR DESCRIPTION
**Pull request by GabyCT**
This script allows to build the cache for kernel experimental.

Fixes #1973

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/install_kata_kernel.sh [hunk 1]
```diff
@@ -132,8 +137,11 @@ main() {
 			build_and_install_kernel
 		fi
 	else
-		build_and_install_kernel
-
+		if [ "$cached_kernel_version" == "$current_kernel_version" ] && [ "$kernel_arch" == "x86_64" ]; then
```
Comment (chavafg) [inline .ci/install_kata_kernel.sh]: I think these lines are not needed as they run the same functions as above. Also I think we can remove the `if` on L128 as you are already taking into consideration the `experimental_kernel` flag inside the `build_and_install_kernel` and `install_cached_kernel` functions.

**Comment by GabyCT** [comment]
@chavafg changes applied

**Comment by GabyCT** [comment]
@chavafg changes applied

**Pull request merged**